testharness/build.gradle.kts

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:Suppress("UnstableApiUsage")
+
+plugins {
+    id(libs.plugins.android.library.get().pluginId)
+    id(libs.plugins.android.kotlin.get().pluginId)
+    id(libs.plugins.jetbrains.dokka.get().pluginId)
+    id(libs.plugins.gradle.metalava.get().pluginId)
+    id(libs.plugins.vanniktech.maven.publish.get().pluginId)
+    jacoco
+}
+
+kotlin {
+    explicitApi()
+}
+
+tasks.withType<Test> {
+    configure<JacocoTaskExtension> {
```

Comment:
I'm not quite sure about this. Are we actually using Jacoco in Accompanist for anything? If I checkout main and run `./gradlew testharness:tasks` I cannot find any tasks relating to Jacoco test coverage. Also doing a search through the codebase only shows Jacoco being referenced in the `:testharness` module if I'm not mistaken. 🤔

Reply:
@alexvanyo any ideas?

Reply:
We checked it when initially creating the module in #1392 via `./gradlew :testharness:createDebugUnitTestCoverageReport`, but it's not monitored and it seems like it doesn't even work properly right now. Removing the configuration here sounds good to me.

Reply:
Alright! Thanks for the answer! I will remove this configuration this :)